examples/lit-ts/.storybook/main.ts

```diff
@@ -0,0 +1,16 @@
+const { mergeConfig } = require('vite');
+const postcssLit = require('rollup-plugin-postcss-lit');
+
+module.exports = {
+  stories: ['../stories/**/*.stories.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  core: {
+    builder: '@storybook/builder-vite',
+  },
+  framework: '@storybook/web-components',
+  async viteFinal(config, { configType }) {
+    return mergeConfig(config, {
+      plugins: [postcssLit()],
```

Comment:
So, it seems that the default `include` from this plugin (`**/*.{css,sss,pcss,styl,stylus,sass,scss,less}`) is breaking with vite, because vite is adding query params to the id.  So, for example, the stylesheet in this PR is becoming `builder-vite/examples/lit-ts/stories/simplegreeting.scss?used`, which is filtered out.  I verified that updating the config as shown below fixes the issue:

```ts
// prettier-ignore
plugins: [postcssLit({ include: ['**/*.scss', '**/*.scss\?*'] })],
```

I had to add the ignore because prettier tries to remove the escape for the `?`.

I think this might be worth opening an issue in https://github.com/umbopepato/rollup-plugin-postcss-lit, if you're up for it.  In the meantime, if you'd like to make the change I suggested above, I'll be happy to move forward with adding this example to our project.

Reply:
Thanks for figuring that out, and so quickly too! I can confirm this worked for us and I've opened an issue in the plugin repo about it.